### ISSUE TITLE
Unable to change Windows orientation

### ISSUE THREAD

**Issue by frankdavidcorona** [pop-os/shell#540]
**(1) Issue/Bug Description:**
I already have two (2) monitors and I'm unable to change Windows orientation pressing **Meta + o** (default keybinding)

**(2) Steps to reproduce (if you know):**
Press **Meta + o** keybindings (default keybinding)

**(3) Expected behavior:**
Windows orientation should change

**(4) Distribution (run `cat /etc/os-release`):**
```
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os
```

**(5) Gnome Shell version:**
3.36

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**
```
pop-shell:
  Instalados: 0.1.0~1599583364~20.04~017c92e
  Candidato:  0.1.0~1599583364~20.04~017c92e
  Tabla de versión:
  0.1.0~1599583364~20.04~017c92e 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu focal/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/pop/ubuntu focal/main i386 Packages
        100 /var/lib/dpkg/status
```

**(7) Where was Pop Shell installed from:**
Official repositories from distribution

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**
1x 1080p (Secondary, Horizontal), 1x 1440p (Primary, Horizontal)

**(9) Other Installed/Enabled Extensions:**
Dash to Panel
Panel OSD
User Themes

**(10) Other Notes:**
In some cases works, but on open e new windows instance or something more related to the Shell, stop working.

**Comment by mmstick** [comment]
Needs reproducible instructions

**Comment by frankdavidcorona** [comment]
> Needs reproducible instructions

Exactly as I said, Open some instances from a window and try: Press Meta + o keybindings (default keybinding)
This is weird behavior, due to as I said before sometimes works and some times didn't work

**Comment by hovo1990** [comment]
I have also faced this issue, but I also don't understand what is causing the issue. I also have a dual monitor setup. Most of the time the only apps I am running are Firefox, Chrome, Pycharm, and terminator. In some cases, It works and in others does not.

**Comment by hovo1990** [comment]
So I just tried to experiment.

1. Performed a reboot, then opened terminator and alacrity, changing window orientation did not work.

2. Then performed another reboot, again opened terminator and alacrity, changing window orientation worked.

3. Then performed another reboot, opened firefox and terminator, changing window orientation continued to work.

4. Then performed another reboot, again opened terminator and alacrity, changing window orientation continued to work.

So still no idea how to reproduce, it is kinda random.

**Comment by frankdavidcorona** [comment]
> So I just tried to experiment.
> 
> 1. Performed a reboot, then opened terminator and alacrity, changing window orientation did not work.
> 2. Then performed another reboot, again opened terminator and alacrity, changing window orientation worked.
> 3. Then performed another reboot, opened firefox and terminator, changing window orientation continued to work.
> 4. Then performed another reboot, again opened terminator and alacrity, changing window orientation continued to work.
> 
> So still no idea how to reproduce, it is kinda random.

Exactly, the same scenario for me. Some times works and some time not

**Comment by mmstick** [comment]
Thus far, I haven't been able to reproduce this issue

**Comment by laichiaheng** [comment]
I have this problem sometimes.

**Comment by jbytes1027** [comment]
happened on two fresh installs - fixed it by removing the keyboard shortcut (super+o) in gnome settings and then re-adding it

**Comment by StarpTech** [comment]
Same issue and solution like @laichiaheng. Sometimes it works sometimes not.

**Comment by Rysher** [comment]
I have a single monitor setup. pop 21.04 COSMIN GNOME version 3.38.5.
This problem also happens to me at random. sometimes right after logging in after a reboot, and sometimes a significant amount of time afterwards. no perticular program involved, it doesn't matter what software I use. @laichiaheng said that super+enter and then enter solves the problem - he is right, it works for me too...until the next time it happens
